Dockerfile

```diff
@@ -7,6 +7,8 @@ FROM nvidia/cuda:10.0-cudnn7-devel-ubuntu18.04
 
 # Get basic packages
 RUN apt-get update && apt-get install -y --no-install-recommends \
+        apt-utils \
+        ca-certificates \
```

Comment:
remove this, `ca-certificates` is already here.

Reply:
done , i missed the other one down below